app/reschedule/template.hbs

```diff
@@ -1,54 +1,66 @@
 <h1>Reschedule</h1>
 
-{{#form-booking-filter page=page as |data|}}
-  {{#paginated-table
-    pages=data.reports.lastSuccessful.value.meta.pagination.pages
-    page=page
-    data-test-search-results=''
-    }}
-    <thead>
+{{report-filter
+  billingTypes  = billingTypes
+  initial       = (hash
+    customer    = _customer
+    project     = _project
+    task        = _task
+    user        = _user
+    reviewer    = _reviewer
+    billingType = billing_type
+    from        = _from
+    to          = _to
+    review      = review
+    notBillable = not_billable
+    notVerified = not_verified
+  )
+  on-apply      = (action 'applyFilter')
+}}
+
+{{#paginated-table
+  data-test-filter-results
+  pages = model.meta.pagination.pages
+  page  = page
+}}
+  <thead>
+    <tr>
+      <th width="4%">User</th>
+      <th width="6%">Date</th>
+      <th width="4%">Duration</th>
+      <th width="10%">Customer</th>
+      <th width="10%">Project</th>
+      <th width="10%">Task</th>
+      <th width="30%">Comment</th>
+      <th width="8%"></th>
+      <th width="8%"></th>
+      <th width="6%"></th>
+      <th width="4%">
+        {{#if model.length}}
+          <button data-test-verify-page class="btn btn-success" title="Verify page" {{action 'verifyPage'}}>{{fa-icon 'check'}}</button>
+        {{/if}}
+      </th>
+    </tr>
+  </thead>
+{{^}}
+  <tbody>
+    {{#each model as |report|}}
+      {{report-reschedule-row report
+        data-test-reschedule-report
+        data-test-reschedule-report-id = report.id
+        verifyUser                     = currentUser
+        on-save                        = (route-action 'saveReport')
+      }}
+    {{^}}
       <tr>
-        <th>Date</th>
-        <th>Dur.</th>
-        <th>Customer / Project / Task</th>
-        <th>Flags</th>
-        <th>User</th>
-        <th>Comment</th>
-        <th>Action</th>
+        <td colspan="11" class="text-center">
+          {{#if (is-array model)}}
+            I didn't find any results - try broadening your filters!
```

Comment:
Not sure if "broaden filters" is something an english-speaking person would say. I've heard of "broaden search", and i could imagine that "loosen filters" or "changing filters" would also be ok.